concent_api/verifier/utils.py

```diff
@@ -96,3 +96,36 @@ def delete_file(file_path):
             os.unlink(file_path)
     except OSError as exception:
         logger.warning(f'File with path {file_path} was not deleted, exception: {exception}')
+
+
+def upload_file_to_storage_cluster(file_content, file_path, upload_token):
```

Comment:
this function is almost identical to the one defined in: `api-integration-force-get-task-result-test`. Please extract it to `utils` module and reuse here and in `api-integration-force-get-task-result-test`

Reply:
:+1: done